PathPlanning/RRT/rrt_with_sobol_sampler.py

```diff
@@ -28,6 +28,7 @@
 
 import math
 import random
+import sys
```

Comment:
Thank you for your PR. Is it possible to add a test to run this script?

Reply:
https://github.com/AtsushiSakai/PythonRobotics/blob/master/tests/test_rrt_with_sobol_sampler.py

Reply:
OK…. There is, I’m no sure why the test did not detect the missing import.

Reply:
OK. This code will be used only when path drawing. Make sense